### ISSUE TITLE
[Bug]: Should macOS Continuous builds use the "under construction" icon?

### ISSUE THREAD

**Issue by justinclift** [sqlitebrowser/sqlitebrowser#3769]
### What did you do?

Trying out the DB4S Nightly and Continuous builds on macOS just now.  The icon for the application is the "under construction" one for the Nightly, but the standard one for the Continuous build.

### Nightly build

![image](https://github.com/user-attachments/assets/95946ff9-70c9-4277-9fbe-6df76e8a6386)

### Continuous build

![image](https://github.com/user-attachments/assets/8302db70-f21f-4e50-905e-9cd16b48f0e9)

Am kind of thinking it should be the "under construction" one for both?

The macOS volume name for both is also saying "Nightly" build too.  We might want to change that?

### What did you expect to see?

-

### What did you see instead?

-

### DB4S Version

3.13.0

### What OS are you seeing the problem on?

MacOS

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by lucydodo** [comment]
Yes, it seems like a good idea to change it. I'll look into it after we finish working on this release process. :)
